cellprofiler/modules/measureobjectintensity.py

```diff
@@ -266,65 +201,59 @@ def upgrade_settings(
                 + setting_values[num_imgs + 1 :]
             )
             variable_revision_number = 3
+        if variable_revision_number == 3:
+            assert not from_matlab
+            num_imgs = int(setting_values[0])
+            images_list = setting_values[1:num_imgs+1]
+            objects_list = setting_values[num_imgs+1:]
+            setting_values = [images_list, objects_list]
+            variable_revision_number = 4
+        else:
+            # Convert saved image and object lists back into python lists
```

Comment:
@0x00b1 @bethac07 - I understand that this is a hacky way of getting a CellProfiler setting to store/recall a list rather than a string. Is there a better way to do this? I expect I'm missing something obvious.